src/schema/schema.graphql

```diff
@@ -2684,6 +2703,7 @@ type Mutation {
 	resetPassword(password: String!): Boolean
 	revokeRights(input: RevokeRightsInput!): RevokeRightsResponse
 	saveDraft(message: SendMessageInput!): SaveDraftResponse
+	saveMessagesToLocal(messagesData: SaveMessagesToLocalInput!): SaveEmailsToLocalResponse
```

Comment:
isn't this same as movemail operation? do we need extra mutation just for local folders?

Reply:
removed